vtex/faststore
# chore: move custom files from customizations/ to customizations/src/

## What's the purpose of this pull request?

This PR looks very scary, but it's essentially only moving things around. It shouldn't change any behavior. It changes where the starter files are copied so it opens room for more complex workflows.

## How it works?

Before, starter files in the `src` folder were copied to the build folder's `src/customizations`. It didn't leave room for root files, such as the config file, to be copied and preserve the folder structure present in the starter.

This PR changes that. Now, starter files in the `src` folder are copied to `src/customizations/src` in the build folder, which allows for root files (such as the config) referencing `./src/*` paths to resolve the appropriate files.

This is used in the PR #2071
## How to test it?

Use the package of this PR in the starter, locally. Make sure everything worked as before. It might help if you start from #2069 's deploy preview.

## References

#2069
